### DISCUSSION TITLE
Cost and usage data not showing up

### DISCUSSION THREAD

**Discussion by ptortiz** [aws-solutions/workload-discovery-on-aws#522] [Q&A]
All resources for all accounts appear are available in the application, but not the cost and usage data. There are also no objects in the cost and usage buckets in S3. Is there something additional that I need to do to enable this or something I missed in the setup? Screenshots are for reference. Thanks.

Paul

![image](https://github.com/aws-solutions/workload-discovery-on-aws/assets/25441522/70f99b12-09d9-4008-827a-2c94ed3cef1f)
![image](https://github.com/aws-solutions/workload-discovery-on-aws/assets/25441522/768e409f-b50c-458b-907f-0fa60578da2d)
![image](https://github.com/aws-solutions/workload-discovery-on-aws/assets/25441522/323b8d5c-9bd2-4c92-8c59-795a033b2f18)

**Comment by svozza** [comment]
The cost feature is not enabled by default. The instructions are in the implementation guide: https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/set-up-the-cost-feature.html.

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
